magic_build_and_package.sh

```diff
@@ -47,6 +52,19 @@ fi
 echo '***** Building *****'
 ./mach build
 
+if [ "$CQZ_INJECT_LOGGING" == "1" ]; then
+  echo '***** Injecting Logging *****'
+  cd ..
+  loggerConfigFile=cliqz-logger-config-mac.json
```

Comment:
What about Linux?

Reply:
> What about Linux?

I believe both MacOS and Linux belong to UNIX-like systems. So mac config will go well for Linux as well

Reply:
Is it possible to maybe disable this mechanism completely for Linux?

Reply:
> Is it possible to maybe disable this mechanism completely for Linux?

Yes, we will disable that for Linux completely.